code/js/controllers/SpotifyController.js

```diff
@@ -24,7 +24,7 @@
     var that = this;
     var selectorIndex;
 
-    if (window.location.hostname === "open.spotify.com") {
+    if (window.location.hostname === "open.spotify.com" || window.location.hostname === "play.spotify.com") {
```

Comment:
@berrberr What's your status on this? I know my solution isn't perfect, but it works for now. You're harvesting some bad reviews on the chrome webstore. I have to advise friends how to use my fork in developer mode instead of simply installing your magnificent extension from the chrome web store.